wallet/src/de/schildbach/wallet/ui/MainActivity.kt

```diff
@@ -88,6 +91,12 @@ class MainActivity : AbstractBindServiceActivity(), ActivityCompat.OnRequestPerm
     private val config: Configuration by lazy { walletApplication.configuration }
     private var fingerprintHelper: FingerprintHelper? = null
     private var retryCreationIfInProgress = true
+    private var activateReceiver = object : BroadcastReceiver() {
```

Comment:
this could be `val` instead of `var`

Reply:
correct, but as it turns out we won't be using BroadcastReceiver after all!

Reply:
fixed